src/clr-angular/data/datagrid/built-in/filters/datagrid-numeric-filter.ts

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { AfterViewInit, Component, ElementRef, EventEmitter, Input, Output, ViewChild } from '@angular/core';
+
+import { ClrDatagridFilter } from '../../datagrid-filter';
+import { ClrDatagridNumericFilterInterface } from '../../interfaces/numeric-filter.interface';
+import { CustomFilter } from '../../providers/custom-filter';
+import { FiltersProvider, RegisteredFilter } from '../../providers/filters';
+import { DomAdapter } from '../../../../utils/dom-adapter/dom-adapter';
+import { DatagridFilterRegistrar } from '../../utils/datagrid-filter-registrar';
+import { DatagridNumericFilterImpl } from './datagrid-numeric-filter-impl';
+import { ClrCommonStrings } from '../../../../utils/i18n/common-strings.interface';
+
+@Component({
+  selector: 'clr-dg-numeric-filter',
+  providers: [{ provide: CustomFilter, useExisting: DatagridNumericFilter }],
+  template: `
+        <clr-dg-filter [clrDgFilter]="registered" [(clrDgFilterOpen)]="open">
+            <!--
+                Even though this *ngIf looks useless because the filter container already has one,
+                it prevents NgControlStatus and other directives automatically added by Angular
+                on inputs with NgModel from freaking out because of their host binding changing
+                mid-change detection when the input is destroyed.
+            -->
+            <input #input_low type="number" name="low" [(ngModel)]="low" *ngIf="open"
+                (keyup.enter)="close()" (keyup.escape)="close()" style="width: 78px"
+                [placeholder]="commonStrings.min"/>
+                <span class="datagrid-filter-input-spacer"></span>
+            <input #input_high type="number" name="high" [(ngModel)]="high" *ngIf="open"
+                (keyup.enter)="close()" (keyup.escape)="close()" style="width: 78px"
```

Comment:
Can we rework the styling so there isn't an explicit width set on the input? That would move some CSS into the datagrid file. I would rather see us have something like a flex element that can scale in case popover sizes are ever changed and it surprises me that we don't have the `clr-input` class on here either for the styling.

Reply:
I moved the fixed width to the CSS.  I don't know much about design -- should we have breakpoints for this now or is that a TODO?

Reply:
No breakpoint at the moment, just would like to get the style attribute out from here.